SkietbaanBE/SkietbaanBE/Controllers/UserController.cs

```diff
@@ -60,12 +60,10 @@ public async Task<IActionResult> AddUser([FromBody] User user)
                 return new BadRequestObjectResult("user cannot be null");
             }
         }
-        // PUT: api/User/
+        // PUT: api/User
         [HttpPut("{id}")]
         public async Task<IActionResult> UpdateUser(int id, [FromBody] User user)
```

Comment:
All the issues @DreyerChris  raised in the PutUserMember controller applies here as well.

Reply:
Its done